src/analyse.py

```diff
@@ -45,12 +45,12 @@
 }
 
 
-def analyse_from_full_xml(xml_string, restrict_to_themes):        
+def analyse_from_full_xml(xml_string, restrict_to_themes, exclude_deprecated):        
```

Comment:
For safety I'd give a default value everywhere a new argument has been added. This and line 90 are the two cases I can see

Reply:
yeah sounds right, will do